### PR TITLE
Added Barracuda NG firewall OS detection

### PR DESCRIPTION
**Pull request by cschoonover91**
Issue #3437

### CODE REVIEW

**Comment by Rosiak** [comment]
Just a very tiny thing, could you please remove the 2 line breaks in the definitions file?
Other than that, it looks ok!

File: includes/discovery/os/barracuda.inc.php [hunk 1]
```diff
@@ -17,4 +17,8 @@
     if (stristr($sysDescr, 'Barracuda Spam Firewall')) {
         $os = 'barracudaspamfirewall';
     }
+    if (stristr($sysDescr, 'Barracuda Firewall')) {
+        $os = 'barracudangfirewall';
```
Comment (laf) [inline includes/discovery/os/barracuda.inc.php]: We have no $os definition for barracudangfirewall.

Full support will need to be added as per: http://docs.librenms.org/Support/FAQ/#faq8

**Comment by laf** [comment]
Please feel free to re-submit this once you've resolved the issues.

**Comment by Rosiak** [comment]
Looks ok, all clear.

**Pull request merged**